.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
 /config
+/rest.http
```

Comment:
What is this used for?

Reply:
its just an alternative of postman for testing purpose in vs code.